tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf

```diff
@@ -107,6 +107,7 @@ Class.Pipeline."deepbuffer-playback" {
 			priority	0
 			# enable lp mode
 			lp_mode		1
+			kcps		50000
```

Comment:
Both this value and the 400k value need to come from a platform file. e.g.
```
include(platform.conf)
kcps PLATFORM_DEEPBUFFER_KCPS
```

Reply:
Not sure why @lgirdwood, if you have processing for the deep buffer pipeline, that can be entirely dependent on the module that's added on this pipeline. Different algorithms might have different requirements, there's no good default value that work for a given platform across all algorithms or configurations.

Reply:
It should only be overridden in the machine topology file where it can be used to adjust machine specific pipelines (aligns with Windows use case now).